src/app.jsx

```diff
@@ -513,6 +514,20 @@ class Application extends React.Component {
                 });
     }
 
+    getUserLingeringEnabled(uid) {
+        const client = cockpit.dbus("org.freedesktop.login1");
+        client.call("/org/freedesktop/login1", "org.freedesktop.login1.Manager", "GetUser",
+                    [uid], { type: "u" })
+                .then(([userpath]) => {
+                    client.call(userpath, "org.freedesktop.DBus.Properties", "Get",
+                                ["org.freedesktop.login1.User", "Linger"], { type: "ss" }).then(([result]) => {
```

Comment:
Please always break the line before .then, so that the .catch aligns and this becomes readable.